README.md

```diff
@@ -75,26 +75,25 @@ variables [here](#configuration-and-environment-variables).
 *  `PCS_AZUREMAPS_KEY` - the [Azure Maps](https://azure.microsoft.com/services/azure-maps/) 
   API Key. This can be set to "static" if you do not have one.
 
-## Running the service with Visual Studio
+## Running the service with Visual Studio or VS Code
+
```

Comment:
should this be two separate sections? Since there are different pre-reqs for visual studio and vs code

Reply:
Ug, maybe. I was following one of the other repos that grouped them together.  That's where I started with this whole task... across a gazillion repos.  I'd prefer to leave them together for now.  When the repos are merged, it'll be so much easier to change it then if we still think they shoudl be separate.